src/redux/greetings/greetingsSlice.js

```diff
@@ -0,0 +1,43 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+
+export const initialState = {
+  greetings: [],
+  isLoading: false,
+  errors: null,
+};
+const URL = 'http://127.0.0.1:3000/api/v1/greetings';
+export const getGreetings = createAsyncThunk('greetings/getGreetings',
+  async () => {
+    try {
+      const response = await fetch(URL);
+      const data = await response.json();
+      return data;
+    } catch (error) {
+      return error.message();
+    }
+  });
```

Comment:
- [ ] I am not able to fetch greetings to the frontend I suggest that you add CORS to the backend